src/sqlittaaja/checker.py

```diff
@@ -97,6 +98,13 @@ def remove_extra_spaces(string):
     return re.sub(pattern, " ", string, flags=re.UNICODE)
 
 
+def remove_dot_commands(string):
+    """Removes lines starting with . [dot] from a string."""
```

Comment:
Remember to add extra empty line after doc-comments.

Reply:
And btw, this could be done in a single line something like this:
```python
return "\n".join(filter(lambda line: not line.startswith("."), sring.split("\n")))
```
if you want to 😉.